test/service.tests.ps1

```diff
@@ -0,0 +1,9 @@
+param(
+    $ServiceUrl
+)
+
+Describe "Service" {
+    It "Should return a healthy response on the healthcheck api" {
+        (Invoke-RestMethod "$ServiceUrl/api/v2/healthcheck").status | Should -Be "ALIVE"
```

Comment:
Check this against the status code of the response instead of payload - i.e. should be <400.

Reply:
Have refactored the test to use this instead. Thanks for highlighting

Reply:
Assumed that referencing a 200 directly is suitable for this purpose but happy to change if required